installer/templates/new/config/dev.exs

```diff
@@ -11,7 +11,9 @@ config :<%= application_name %>, <%= application_module %>.Endpoint,
   debug_errors: true,
   code_reloader: true,
   check_origin: false,
-  watchers: <%= if brunch do %>[node: ["node_modules/brunch/bin/brunch", "watch", "--stdin"]]<% else %>[]<% end %>
+  watchers: <%= if brunch do %>[node: ["node_modules/brunch/bin/brunch", "watch", "--stdin",
+                    cd: Path.expand("../", __DIR__)]]<% else %>[]<% end %>
```

Comment:
nitpick: are you aligning this with `node: ` or the first list element? because it should totally be the first list element. :P